recipes-openxt/xenclient-root-ro/xenclient-root-ro/init.root-ro

```diff
@@ -337,6 +361,16 @@ unlock_config()
                 touch ${measured_flag}
                 return 0
             }
+
+            if [ -e ${tss_path}.srtm ]; then
+                maybe_break "srtm-upgrade"
+                rm ${tss_path}.srtm
```

Comment:
Shouldn't the flag be removed later? If for example the user is instructed to shutdown and call an admin when dialog messages happen at boot time, which would make sense, the admin will not see the message again next boot. But maybe that's the desired behavior here...

Reply:
Honestly the flag approach seems so flimsy I'd rather just release note the upgrade behavior for this case and decline this PR.